pegasus/sites.v3/code.org/public/educate/csp/unit4.moved

```diff
@@ -0,0 +1 @@
+https://curriculum.code.org/csp/unit4v1/
```

Comment:
Units 4-6 are actually still temporary. I set a reminder to update these again to /csp/unit4 when those pages are launched (in a couple of weeks).